src/main/anovos/data_analyzer/stats_generator.py

```diff
@@ -493,7 +493,14 @@ def measures_of_centralTendency(
         summary_lst.append(summary_col)
     summary_df = spark.createDataFrame(
         summary_lst,
-        schema=("key", "mean", "50%", "count"),
+        schema=T.StructType(
+            [
+                T.StructField("key", T.StringType(), True),
+                T.StructField("mean", T.DoubleType(), True),
+                T.StructField("50%", T.DoubleType(), True),
+                T.StructField("count", T.LongType(), True),
```

Comment:
This change is reasonable to fix the bug you observed. However, strictly assign the type for `key`, `mean`, `50%`, and `count` will cause some format error. Refer to https://github.com/anovos/anovos/actions/runs/3038577821/jobs/4893029204#step:11:187 for the error log.